packages/manager/apps/container/src/shell/index.jsx

```diff
@@ -30,11 +30,26 @@ function Shell() {
 
   useEffect(() => {
     const routing = plugin.routing.initRouting(iframeRef.current);
+
+    // Hub application redirections
     routing.addRoute(
       <Route exact path="/catalog">
         <Redirect to="/hub/catalog" />
       </Route>,
     );
+
+    // Telecom application redirections
+    [
+      'freefax/:id?',
+      'pack/:id?',
+      'sms/:id?',
+      'telephony/:id?',
```

Comment:
We forgot over the box I think

Reply:
and telecom tasks & orders (some /orders routes are exposed)